commands/utilities/say.js

```diff
@@ -0,0 +1,58 @@
+const { Command } = require('discord.js-commando');
+const { MessageEmbed, MessageManager } = require('discord.js');
+
+module.exports = class sayCommand extends Command {
+  constructor(client) {
+    super(client, {
+      name: 'say',
+      aliases: ['announce'],
+      group: 'utilities',
+      memberName: 'say',
+      description: 'Makes a server announcement',
+      examples: ['!say "title" "announcement"'],
+      guildOnly: true,
+      clientPermissions: ['SEND_MESSAGES'],
+      hidden: true,
+      args: [
+        {
+          key: 'title',
+          prompt: 'What is the announcement title?',
+          type: 'string',
+        },
+        {
+          key: 'announcement',
+          prompt: 'What is the announcement?',
+          type: 'string',
+        },
+      ],
+    });
+  }
+
+  hasPermission(message) {
+    const approvedRoles = ['⚔️ Commander'];
+    const title = message.member.roles.highest.name;
+    if (approvedRoles.includes(title)) return true;
+    return 'Only Division Commanders and above may create events.';
```

Comment:
Incorrect

Reply:
Permission are copied from another command so I'll have to go through them. I want to change the categories too so there help command makes it clear who can use what commands